tools/privacyidea-token-janitor

```diff
@@ -400,6 +406,7 @@ def export_user_data(token_list):
                               'the output should only contain the number of tokens owned '
                               'by the user.',
                 dest='sum_tokens', action='store_true')
+@manager.option('--attributes', help='Extends the sum function to display additional user attributes.')
```

Comment:
would be great to do

     Extends the "listuser" function to display additional user attributes

(see below.)